docs/pull-request-faq.md

```diff
@@ -0,0 +1,37 @@
+# Yeti Pull Request FAQ
+
+This document addresses common questions that come up when you are ready to create a pull request and submit your work.
+
+## What are the PR template sections for?
+
+**Ticket**
+
+- Provide a link to the work ticket so that the reviewer can easily see the task requirements.
+
+**Changes**
+
+- A high-level summary list of the changes helps the reviewer to quickly understand the purpose of the code.
+
+**Notes (optional)**
+
+- This is a place to add context if you feel it would be helpful. Some examples would be to outline any future related work that is still needed, describe why an important decision was made regarding the approach, or explain why a particular piece isn't working yet and what the plan is to resolve it.
+
+**Visuals**
+
+- Show that this is working as expected and meets the requirements by uploading a screenshot or demo video. In most cases, the reviewer should not need to test your branch locally to determine this.
+
+## How should I organize or break up my PRs?
+
+Different developers may have different preferences for this. In general, it's a good idea to group the contents of a PR into a set of changes that will achieve a specific verifiable functionality. This allows the reviewer to better understand how things are working since all of the changes that depend on each other are submitted together. This only holds to a certain point - if the diff becomes very large, you should consider breaking it up into more manageable pieces.
+
+## Who should I tag as a reviewer?
+
+For smaller projects, it's typically best to include all developers that are working in the repository. This is so that each person has visibility into other parts of the application that could affect their work directly or indirectly. If you are working on a project as a solo developer, you should arrange in advance to have another Yeti team member added to the repo who can provide at minimum a sanity check on your changes and preferably add useful feedback.
+
+## How long should I wait for a review?
+
+In most cases, it's best to allow a reviewer up to 24 hours to approve your work. If you have multiple reviewers on a PR, there should be a designated person, usually the tech lead for the project, who's review is required. If the lead developer approves your changes, it's assumed that you can go ahead with the merge.
```

Comment:
` If the lead developer approves your changes, it's assumed that you can go ahead with the merge.` Is this how we usually go about things? My experience has been that we generally wait for 2 reviews unless the feature is very small/needs to be rushed

Reply:
I think what I meant to say didn't come across very well here. I was thinking that as long as the lead dev has approved after 24 hours, it's okay to merge.

It feels like we're handling this a bit differently depending on the dev team or project. We should probably discuss this some more